api/api.go

```diff
@@ -2,6 +2,7 @@ package api
 
 import (
 	"encoding/json"
+	"gopkg.in/cas.v2"
```

Comment:
This should be moved down a bit to separate it from the standard library packages.

Reply:
gofmt puts it here

Reply:
Check out https://godoc.org/golang.org/x/tools/cmd/goimports

We keep them separate in other import statements, and I'd like to continue